tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why is this separate from `tsconfig-base.json`? Seems like this one is not connected to the rest of the configs.

Reply:
`tsconfig.json` is the root of the project tree.

`tsconfig-base.json` are just common ts-configurations shared between the configs.

You could copy the contents of `tsconfig-base` into each of the `tsconfigs` to see the same result.